sdk/monitor/Azure.Monitor.Ingestion/CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
+Version 1.0.0-beta.1 is a preview of our efforts in creating a client library for Azure Logs Ingestion that is
+developer-friendly, idiomatic to the .NET ecosystem, and as consistent across different languages and platforms as
+possible. The principles that guide our efforts can be found in the
+[Azure SDK Design Guidelines for .NET](https://azure.github.io/azure-sdk/dotnet_introduction.html).
+
+## Features Added
+- Initial release. Please see the README and wiki for information on using the new library.
```

Comment:
No idea why the template includes that because we don't have a wiki.  I'm guessing this came from some other language...

```suggestion
- Initial release. Please see the README for information on using the new library.
```